site/src/pages/DeploymentSettingsPage/IdpOrgSyncPage/IdpOrgSyncPageView.tsx

```diff
@@ -0,0 +1,349 @@
+} from "components/TableLoader/TableLoader";
+import { Button } from "components/ui/button";
+import { Input } from "components/ui/input";
+import { Label } from "components/ui/label";
+import MultipleSelector, { type Option } from "components/ui/multiple-selector";
+import { Switch } from "components/ui/switch";
+import { useFormik } from "formik";
+import { Plus, SquareArrowOutUpRight, Trash } from "lucide-react";
+import type React from "react";
+import { useState } from "react";
+import type { FC } from "react";
+import { MONOSPACE_FONT_FAMILY } from "theme/constants";
+import { docs } from "utils/docs";
+import { PillList } from "./PillList";
+
+interface IdpSyncPageViewProps {
+	organizationSyncSettings: OrganizationSyncSettings | undefined;
+	organizations: readonly Organization[];
+	onSubmit: (data: OrganizationSyncSettings) => void;
+	error?: unknown;
+}
+
+export const IdpSyncPageView: FC<IdpSyncPageViewProps> = ({
+	organizationSyncSettings,
+	organizations,
+	onSubmit,
+	error,
+}) => {
+	const form = useFormik<OrganizationSyncSettings>({
+		initialValues: {
+			field: organizationSyncSettings?.field || "",
+			organization_assign_default:
+				organizationSyncSettings?.organization_assign_default || true,
+			mapping: organizationSyncSettings?.mapping || {},
+		},
+		// validationSchema,
+		onSubmit,
+		enableReinitialize: true,
+	});
+	const [coderOrgs, setCoderOrgs] = useState<Option[]>([]);
+	const [idpOrgName, setIdpOrgName] = useState("");
+	const [syncSettings, setSyncSettings] = useState(
+		organizationSyncSettings || {
+			field: "",
+			organization_assign_default: true,
+			mapping: {},
+		},
+	);
+
+	const organizationMappingCount = syncSettings?.mapping
+		? Object.entries(syncSettings.mapping).length
+		: 0;
+
+	if (error) {
+		return <ErrorAlert error={error} />;
+	}
+
+	if (!organizationSyncSettings) {
+		return <Loader />;
+	}
+
+	const OPTIONS: Option[] = organizations.map((org) => ({
+		label: org.name,
+		value: org.id,
+	}));
+
+	const getOrgNames = (orgIds: readonly string[]) => {
+		return orgIds.map(
+			(orgId) =>
+				organizations.find((org) => org.id === orgId)?.display_name || orgId,
+		);
+	};
+
+	const handleDelete = async (idpOrg: string) => {
+		const newMapping = Object.fromEntries(
+			Object.entries(syncSettings?.mapping || {}).filter(
+				([key]) => key !== idpOrg,
+			),
+		);
+		const newSyncSettings = {
+			...(syncSettings as OrganizationSyncSettings),
+			mapping: newMapping,
+		};
+		setSyncSettings(newSyncSettings);
+		await form.setFieldValue("mapping", newSyncSettings.mapping);
+	};
+
+	return (
+		<>
+			<Stack spacing={2}>
+				<form onSubmit={form.handleSubmit}>
+					<fieldset disabled={form.isSubmitting}>
+						<div className="flex flex-row">
+							<div className="grid items-center gap-1">
+								<Label className="text-sm" htmlFor="sync-field">
+									Organization sync field
+								</Label>
+								<div className="flex flex-row items-center gap-4">
+									<Input
```

Comment:
Are we moving away from blue focus styling? Just making sure, since I wasn't expecting white

<img width="597" alt="Screenshot 2024-11-22 at 11 03 47 AM" src="https://github.com/user-attachments/assets/27abdd38-7730-4c79-9e6e-9183a21b6e31">

Reply:
I synced with Christin and we will keep the blue focus styling. I updated the relevant components.